app/assets/javascripts/views/question_form.js

```diff
@@ -65,6 +65,23 @@
       $(".form_field#maximum input[id$='_maxstrictly']").prop("checked", false);
     }
 
+    // show/hide key question
+    var hide_key_q = (
+      selected_type == "image" ||
+      selected_type == "annotated_image" ||
+      selected_type == "signature" ||
+      selected_type == "sketch" ||
+      selected_type == "audio" ||
+      selected_type == "video"
+    );
```

Comment:
I think we should bite the bullet and transfer the list of multimedia question types to a JS constant someplace. Or even better you could transfer the full set of QuestionTypes as JSON. They are used all over the place. Then here you could just do .index(..) != -1